### PR TITLE
Assign reviewers to PRs when they approve (take 2)

### PR DESCRIPTION
**Pull request by Wafflespeanut**
adds null check to #92

### CODE REVIEW

**Comment by jdm** [comment]
Can we add a test for the null-check case?

**Comment by Wafflespeanut** [comment]
@jdm Yes, done! :)

File: handlers/assign_reviewer/tests/no-reviewer.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
+{
+  "initial": {
+  },
```
Comment (jdm) [inline handlers/assign_reviewer/tests/no-reviewer.json]: Let's set an initial assignee and ensure that they're still set in the expected values.
